sklearn/mixture/dpgmm.py

```diff
@@ -698,7 +698,11 @@ def __init__(self, n_components=1, covariance_type='diag', alpha=1.0,
             n_components, covariance_type, random_state=random_state,
             tol=tol, verbose=verbose, min_covar=min_covar,
             n_iter=n_iter, params=params, init_params=init_params)
-        self.alpha = float(alpha) / n_components
```

Comment:
Not sure I understand that remark ^^ You mean in fit, not in ``__init__``, right?